### PR TITLE
Work on documenting the generator

### PR DESCRIPTION
**Pull request by BinAccel**
This is all I can recall of the top of my head + reading the `ds3` generator. Someone tell me if I've had too many shrooms

### CODE REVIEW

**Comment by quantum5** [comment]
Generated input data goes through stderr? Are you really sure?

**Comment by BinAccel** [comment]
I remember someone said that, but I was right the first time

File: problem_examples/generator/ds3/README.md [hunk 1]
```diff
@@ -1 +1,5 @@
 # Automatically-Generating Input/Output Data Large Input/Output Data
+
+For some problems, the test data takes up a very large amount of space, and it may not be feasible to store all the files in full. As such, the judge allows for the use of generators.
+
+The judge will invoke a generator when it sees a `generator` node in the `init.yml`. The generator will have acess to the input files specificed in the `test_cases` node. The generated input data is printed through `stdout` and the generated output data is printed through `stderr`.
```
Comment (Xyene) [inline problem_examples/generator/ds3/README.md]: > specificed

File: problem_examples/generator/ds3/README.md [hunk 2]
```diff
@@ -1 +1,5 @@
 # Automatically-Generating Input/Output Data Large Input/Output Data
+
+For some problems, the test data takes up a very large amount of space, and it may not be feasible to store all the files in full. As such, the judge allows for the use of generators.
```
Comment (Xyene) [inline problem_examples/generator/ds3/README.md]: > allows for the use of generators, to generate large data on the fly.

File: problem_examples/generator/ds3/README.md (hunk 1 above)
Comment (BinAccel) [inline problem_examples/generator/ds3/README.md]: Please, lord Dictionary, enlighten me on the proper spelling

Comment (Xyene) [inline problem_examples/generator/ds3/README.md]: > specified

**Pull request merged**